ecosystem/sep-0008.md

```diff
@@ -61,24 +59,21 @@ approval_criteria="The goat approval server will ensure that transactions are co
 ```
 
 ## Approval Server
-The transaction approval server receives a signed transaction, checks for compliance and signs if possible.
+
+The transaction approval server receives a signed transaction and checks for compliance and signs it on success.
 
 ### Possible results
 - Success: Transaction was found compliant and signed by service.
-- Revised: Transaction was modified to be compliant and signed by service. It should be re-signed by the client.
-- Pending: The issuer will asynchronously validate the transaction and respond later.
-- Action Required: The user must complete an action in order to have the transaction approved.
 - Rejected: Transaction was rejected.
 
 ### Request
 
-Transaction approval requests are executed by submitting an `HTTP POST` to `approval_server` with `Content-Type=application/x-www-form-urlencoded` and a single `tx` parameter. `tx` value is a base64 encoded Transaction Envelope XDR signed by the user. This is the transaction that will be tested for compliance and signed if possible.
+Transaction approval requests are executed by submitting an `HTTP POST` to `approval_server` with `Content-Type=application/x-www-form-urlencoded` and a single `tx` parameter. `tx` value is a base64 encoded transaction envelope XDR signed by the user. This is the transaction that will be tested for compliance and signed on success.
```

Comment:
💡 I think we should make the request allowed to be either form urlencoded or JSON encoded. We do this in some other SEPs, e.g. SEP-10, SEP-30. The viewpoint from some people outside SDF that led to this pattern on SEP-10 was that it makes it easier to integrate with the APIs in some cases.

Reply:
https://github.com/stellar/stellar-protocol/pull/750/commits/a8d3bf64b67694086d4b7a2042e08a6b2ed5f265